src/style.css

```diff
@@ -0,0 +1,57 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
```suggestion
```
redundant to reset margin and padding for all selectors.
reset only elements where it is necesary